Sources/MapboxMaps/Annotations/Generated/CircleAnnotationManager.swift

```diff
@@ -41,13 +41,17 @@ public class CircleAnnotationManager: AnnotationManager {
     /// Dependency required to add gesture recognizer to the MapView
     private weak var view: UIView?
 
-    internal init(id: String, style: Style, view: UIView, mapFeatureQueryable: MapFeatureQueryable, layerPosition: LayerPosition?) {
+    /// Indicates whether the style layer exists after style changes. Default value is `true`.
+    public private(set) var shouldPersist: Bool
```

Comment:
I thought you'd changed this to be a `let` and `internal`?

Reply:
Oh, thanks! Switched it back.